pkgs/tools/archivers/p7zip/default.nix

```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation rec {
   patches = [
     ./12-CVE-2016-9296.patch
     ./13-CVE-2017-17969.patch
+    ./CVE-2018-5996.patch
+    ./CVE-2018-10115.patch
```

Comment:
Can we fetch these from here?
https://github.com/termux/termux-packages/tree/master/packages/p7zip

Reply:
Possibly, though those patches aren't byte-identical, they look similar to me. I've just pushed a new version where these are downloaded with fetchpatch.